integration/js/utils/WebdriverSauceLabs.js

```diff
@@ -107,41 +107,33 @@ const getSauceLabsDomain = (platform)  => {
   if (platformUpperCase === 'LINUX') {
     return 'us-east-1.saucelabs.com';
   } else if (platformUpperCase === 'ANDROID' || platformUpperCase === 'IOS') {
-    return 'us1-manual.app.testobject.com';
+    return 'us-west-1.saucelabs.com';
```

Comment:
can it be just saucelabs.com which I think will resolve to us-west-1 anyway?

Reply:
Also is there a way to configure these values instead of hardcoding them in code.

Reply:
@ltrung I had thought this as well, but using simply saucelabs.com results in a capabilities not found error. Not entirely sure why this happens as the other tests being ran using the url seem to go to us-west-1, but it probably has to do with how saucelabs redirects requests. I don't think it would be a bad idea to change the default return to us-west-1 though that way we could get rid of the mobile check right there, what are your thoughts?

Reply:
Yeah if the other test with us-west-1 URL we can do that too :)

Reply:
@harshitOberoi The platform is already configurable, and as for the url this will cause more issues than merits. There are only two servers we use and specific platforms can only be ran on specific servers so making it configurable will introduce room for error in misconfiguration and result in more code in trying to configure each test.

Reply:
@ltrung Changed!